frontend/src/tests/components/commons/SideNavigation.test.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { mount } from "enzyme";
+import SideNavigation from "../../../components/commons/SideNavigation";
+
+const BODY1 = <div>Body 1</div>;
+const BODY2 = <div>Body 2</div>;
+const BODY3 = <div>Body 3</div>;
+const BODY4 = <div>Body 4</div>;
+const TEXT1 = "test1";
+const TEXT2 = "test2";
+const TEXT3 = "test3";
+const TEXT4 = "test4";
+
+const BTN1_SELECTED = "#unit-test-sidenav-button0-selected";
+const BTN1_UNSELECTED = "#unit-test-sidenav-button0-unselected";
+const BTN2_SELECTED = "#unit-test-sidenav-button1-selected";
+const BTN2_UNSELECTED = "#unit-test-sidenav-button1-unselected";
+const BTN3_SELECTED = "#unit-test-sidenav-button2-selected";
+const BTN3_UNSELECTED = "#unit-test-sidenav-button2-unselected";
+const BTN4_SELECTED = "#unit-test-sidenav-button3-selected";
+const BTN4_UNSELECTED = "#unit-test-sidenav-button3-unselected";
+
+it("renders 2 buttons", () => {
+  const SPECS = [{ id: 0, text: TEXT1, body: BODY1 }, { id: 1, text: TEXT2, body: BODY2 }];
```

Comment:
I'm having a hard time reading what all of this means - it's fairly complex to check something pretty simple.
To check if two buttons exist, what if we just said:
`expect(wrapper.find('.btn-primary')).to.have.lengthOf(1);`
`expect(wrapper.find('.btn-secondary')).to.have.lengthOf(1);`

Reply:
Similar for the other tests looking at checking number of buttons.

Reply:
Just as  note, in case anyone refers back to this function is "toHaveLength"; resolved in ce3a402